package.json

```diff
@@ -67,8 +68,8 @@
     "version"
   ],
   "license": "MIT",
-  "main": "index.js",
-  "nyc": {
+  "exports": "./index.js",
+  "c8": {
```

Comment:
nice to see this one. i've had this type of conversion on my list for some projects for a while, but i guess i havent looked deep enough yet to know that the switch was this simple

Reply:
I think it is, it's the same @bcoe behind both projects :)